File: Nian_Karl/QL/src/org/uva/ql/ast/expression/binary/Multiply.java
package org.uva.ql.ast.expression.binary;

import org.uva.ql.ast.builder.CodePosition;
import org.uva.ql.ast.expression.Expression;
import org.uva.ql.visitor.Visitor;

public class Multiply extends Binary {

	public Multiply(Expression left, Expression right, CodePosition pos) {
		super(left, right, pos);
	}

	@Override
	public <T> T accept(Visitor<T> visitor) {
		return visitor.visit(this);
	}
	
	@Override
	public String toString() {
		return this.left.toString() + " * " + this.right.toString();
	}

}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/Expressions/Binary/Multiply.cs
using AST.Nodes.Interfaces;
using AST.Representation;

namespace AST.Nodes.Expressions.Binary
{
    public class Multiply : Binary
    {
        public Multiply(Expression left, Expression right, PositionInText position)
            : base(left, right, position)
        { }

        public override T Accept<T>(ASTVisitors.IVisitor<T> visitor)
        {
           return visitor.Visit(this);
        }

        public override string ToString()
        {
            return "*";
        }
    }
}
